op-verifier/contracts/OPVerifier.sol

```diff
@@ -1,43 +1,177 @@
 // SPDX-License-Identifier: MIT
 pragma solidity ^0.8.17;
 
-import { IEVMVerifier } from "@ensdomains/evm-verifier/contracts/IEVMVerifier.sol";
-import { RLPReader } from "@eth-optimism/contracts-bedrock/src/libraries/rlp/RLPReader.sol";
-import { StateProof, EVMProofHelper } from "@ensdomains/evm-verifier/contracts/EVMProofHelper.sol";
-import { Types } from "@eth-optimism/contracts-bedrock/src/libraries/Types.sol";
-import { Hashing } from "@eth-optimism/contracts-bedrock/src/libraries/Hashing.sol";
+import {IEVMVerifier} from '@ensdomains/evm-verifier/contracts/IEVMVerifier.sol';
+import {RLPReader} from '@eth-optimism/contracts-bedrock/src/libraries/rlp/RLPReader.sol';
+import {StateProof, EVMProofHelper} from '@ensdomains/evm-verifier/contracts/EVMProofHelper.sol';
+import {Types} from '@eth-optimism/contracts-bedrock/src/libraries/Types.sol';
+import {Hashing} from '@eth-optimism/contracts-bedrock/src/libraries/Hashing.sol';
+import './interfaces/IDisputeGame.sol';
```

Comment:
Can't we import from `@eth-optimism/contracts-bedrock` ? the file looks slightly from https://github.com/ethereum-optimism/optimism/blob/develop/packages/contracts-bedrock/src/dispute/interfaces/IDisputeGame.sol

Reply:
I'm facing `Error HH411: The library src, imported from @eth-optimism/contracts-bedrock/src/dispute/interfaces/IDisputeGame.sol, is not installed. Try installing it using npm.` due to they have migrated to use foundry absolute import path.

I have asked at https://github.com/NomicFoundation/hardhat/discussions/5049

> I think this is an issue with the way Hardhat and Foundry resolve Solidity imports and they clash - Foundry resolution is based on remappings, while Hardhat has a treats direct imports as coming from the node_modules folder.
> 
> We are looking to add first class support for remappings to Hardhat to support these sorts of cases.

It sounds like they are implementing this feature but is not supported yet.

Reply:
Did you also try to upgrade `@eth-optimism/contracts-bedrock` to the latest version?

Reply:
I have build another NPM package and use foundry flatten on optimism interfaces

https://github.com/Opti-domains/dispute-game-lookup/blob/main/npm/contracts/interfaces/optimism/IDisputeGameFactory.sol